mobify/progressive-web-scaffold
# Fix hotloader issues, refactor analytics in router

## Changes
- Use latest version of react-hot-loader to fix debugging issues
- Clean up router analytics code

## How to test-drive this PR
### To test Analytics:
- Install the [GA debugger chrome extension](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en)
- Load a page, ensure a GA page view is triggered
- Navigate to a different page, ensure page view is triggered

### To test hotloader fix:
- Add `throw Error('test')` inside the `triggerPageView` function in main.jsx, ensure you can see the error in the browser
